### DISCUSSION TITLE
Create aliases automatically on the fly

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1005] [Feature Requests]
To create aliases on the fly all you have to do is make up any new alias and give that out instead of your real email address.

Let's say you're username is johndoe and you are signing up to example.com you could enter example@johndoe.example.com as your email address.

The alias will show up in your dashboard automatically as soon as it has forwarded its first email.

This works the same way for custom domains and additional usernames, your aliases are created as soon as they receive their first email.

**Comment by notDavid** [comment]
Isn't your suggestion the same as "Auto create/on the fly alias: Catch All" ?  https://simplelogin.io/docs/custom-domain/manage-domain/
